README.md

```diff
@@ -45,11 +45,25 @@ Once your docker is running, you can configure the Wordpress installation:
 For unit testing a separate CLI based wordpress is included, this can be started trough
 
 ```CLI
-$docker compose -f docker-compose-cli.yml up
+$ docker compose -f docker-compose-cli.yml up
 ```
 
 You can than run [Wordpress CLI commands](https://developer.wordpress.org/cli/commands/) with in the container. For example, getting a list of all available plugins can be done trough
 
 ```CLI
-$docker-compose exec wordpress wp plugin list
+$ docker compose exec wordpress wp plugin list
 ```
+
+Activating the plugin (if  it isnt activated yet)
+
+```CLI
+$ docker compose  exec wordpress wp plugin activate OpenPub
+```
+
+## Security
+We use [Grype](https://github.com/anchore/grype) as a scanning tool for containers as a [action](https://github.com/marketplace/actions/anchore-container-scan) in our git workkflow. The flow is wired to fail when an critcal error is found (in wich casses no new release artifacts are created). This can be altered in the test.yamml worklow.  The fail point can be set to `low`,`medium`,`high` or `critical` alternativly the fail point can be ignored completly by setting `fail-build` to `false`. Aditiona configuration for snyk and trivy are also  included (but commented out) and are not recomended.
```

Comment:
We use [Grype](https://github.com/anchore/grype) as a scanning tool for containers as an [action](https://github.com/marketplace/actions/anchore-container-scan) in our git workflow. The flow is wired to fail when a critical error is found (in which case, no new release artifacts are created). This can be altered in the test.yaml workflow. The fail point can be set to `low`,`medium`,`high` or `critical` alternatively the fail point can be ignored completely by setting `fail-build` to `false`. Additional configurations for Snyk and Trivy are also included (but commented out) and are not recommended.